CIME/Tools/Makefile

```diff
@@ -656,6 +655,14 @@ ULIBS += -L$(INSTALL_SHAREDPATH)/lib
 
 ULIBS += -lcsm_share -L$(INSTALL_SHAREDPATH)/lib $(PIOLIBNAME) -lgptl
 
+# Add Kokkos header files and libraries for SE-NH dycore
```

Comment:
I'm wondering if this small make block is sufficient to introduce Kokkos into the Makefile build system? On the E3SM side, we use CMake, so we can just do:
`target_link_libraries($target PUBLIC Kokkos::kokkos)`

and we get everything we need, including whatever build and link flags Kokkos thinks it needs to inject for the current platform.

Reply:
Thanks @jgfouca . Currently we only plan to use kokkos for the `theta-l` and `theta-l_kokkos` dycores in CAM, and I confirm that this block is sufficient.